client/curriculum/html/LinksAndButtons.tsx

```diff
@@ -0,0 +1,882 @@
+  <span style="position: absolute; opacity: 0">
+  Useful link text
+  </span>
+ </a>`}
+          />
+          <LessonFigure
+            withBorder
+            src="/images/lessons/links-and-buttons/icon-inside-link.png"
+            alt="Doar un icon în interiorul unui link"
+            demo="/demo/html/icon-in-link"
+          />
+          <LessonTip icon={faQuestionCircle}>
+            Atributul
+            {' '}
+            <strong className="formatted">style</strong>
+            {' '}
+            este folosit
+            pentru a adăuga reguli CSS elementelor. Încă nu am ajuns la acel
+            capitol deci e absolut normal să nu știi ce face.
+            <br />
+            {' '}
+            <br />
+            Totuși, te rugăm să-l pui acolo, e un exemplu bun de avut în vedere.
+          </LessonTip>
+          <blockquote>
+            Putem folosi o
+            {' '}
+            <strong className="formatted">imagine</strong>
+            {' '}
+            în
+            interiorul unui link?
+          </blockquote>
+          <p>
+            Daaa. Vom avea nevoie de multe ori să facem asta în proiectele
+            noastre.
+          </p>
+          <Highlight
+            className="my-5"
+            language="html"
+            code={`
+<a href="/">
+  <img src="images/logo.png" alt="Frontend.ro LOGO." />
+</a>`}
+          />
+          <LessonFigure
+            withBorder
+            src="/logo.png"
+            alt="O imagine în interiorul unui link"
+            demo="/demo/html/imagine-in-link"
+          />
+          <p>
+            Nu e nevoie să folosim atributul
+            {' '}
+            <strong className="formatted">alt</strong>
+            {' '}
+            pentru a preciza că
+            imaginea este un link, tehnologiile assistive vor face deja asta
+            pentru noi.
+          </p>
+          {/* this demo 👇 is not very clear at this moment, when we will have more CSS lessons,
+           maybe we can uncomment this part */}
+          {/* <blockquote>
+            Putem folosi bucăți mari de cod în interiorul unui link?
+          </blockquote>
+          <p>Răspunsul la această întrebare este că nu e tocmai o idee bună.</p>
+          <LessonFigure
+            withBorder
+            src="/images/lessons/links-and-buttons/demo-link.png"
+            alt="2 metode diferite prin care putem folosi link-urile în raport cu bucăți
+            mai mari de cod și implicațiile lor"
+            demo="/demo/html/linkuri-care-contin-bucati-mari-de-continut"
+          /> */}
+        </section>
+        <section>
+          <LessonHeading as="h3" id="button-element">
+            {'<button> element'}
+          </LessonHeading>
+          <p>
+            Elementul html
+            {' '}
+            <strong className="formatted">{'<button>'}</strong>
+            {' '}
+            este folosit atunci când vrem să facem o acțiune (exemple uzuale de
```

Comment:
Remove "exemple uzuale de actiuni" and keep just "exemple". You just mentioned actions.